app/services/bulk_upload/processor.rb

```diff
@@ -1,15 +1,22 @@
 class BulkUpload::Processor
+  include CollectionTimeHelper
   attr_reader :bulk_upload
 
-  BLANK_TEMPLATE_ERRORS = [
-    I18n.t("activemodel.errors.models.bulk_upload/lettings/validator.attributes.base.blank_file"),
-    I18n.t("activemodel.errors.models.bulk_upload/sales/validator.attributes.base.blank_file"),
-  ].freeze
+  def blank_template_errors
+    [
+      I18n.t("validations.lettings.#{current_collection_start_year}.bulk_upload.blank_file"),
+      I18n.t("validations.lettings.#{previous_collection_start_year}.bulk_upload.blank_file"),
+      I18n.t("activemodel.errors.models.bulk_upload/sales/validator.attributes.base.blank_file"),
+    ].freeze
```

Comment:
Is there / will there be a similar thing for sales here?

Reply:
Yes there was